File: postgre/lua/delete/postgre_delete.lua
-- ----------------------------------------------------
--	postgre_delete.lua
--
--					Jul/16/2011
-- ----------------------------------------------------
require	("luasql.postgres")
require	("sql_manipulate")

print ("*** 開始 ***")
local id_in= arg[1]

print (id_in)

local dbname = "city"
local dbhost = "localhost"
local dbuser = "scott"
local dbpass = "tiger"

local env = assert(luasql.postgres())
local con = assert(env:connect(dbname, dbuser, dbpass, dbhost))

sql_delete_proc (con,id_in)

con:close()
env:close()

print ("*** 終了 ***")
-- ----------------------------------------------------

File: postgre/typescript/delete/postgre_delete.ts
// ---------------------------------------------------------------
//	postgre_delete.ts
//
//					Feb/09/2016
//
// ---------------------------------------------------------------
declare function require(x: string): any;
declare var process:any;

var pg = require('pg');
var sql_manipulate= require ('/var/www/data_base/common/node_common/sql_manipulate');

// ---------------------------------------------------------------
console.log ("*** 開始 ***");
var id_in: string = process.argv[2];

console.log (id_in);

var str_connect: string = "tcp://scott:tiger@localhost:5432/city";

pg.connect (str_connect,function (error,client)
	{
	var command: string = sql_manipulate.delete_command_gen (id_in);

	console.log (command);

	var query = client.query(command);

/*
	query.on('row',function (row) {
		});
*/

	query.on('end',function () {
		client.end ();
		});

	console.log ("*** 終了 ***");
	});

// ---------------------------------------------------------------
